test/node/animas/testAnimasSimulator.js

```diff
@@ -493,7 +493,7 @@ describe('animasSimulator.js', function() {
         type: 'deviceEvent',
         subType: 'status',
         status: 'suspended',
-        reason: {suspended: 'automatic', resumed: 'automatic'},
+        reason: {suspended: 'automatic', resumed: 'manual'},
```

Comment:
@jebeck Just want to make sure I'm using the nomenclature for `reason` correctly here. For the fabricated suspend/resume events I'm marking `suspended` as `automatic` (since the alarm initiated the suspend), but I mark `resumed` as `manual`, as the user needs to perform some action before the pump will resume (even though it does not necessarily mean clicking a "resume" button).